apps/docs/src/guide/types/bits512.md

```diff
@@ -4,22 +4,26 @@ In Sway, the `b512` type is commonly used to handle public keys and signatures.
 
 The `b512` type in Sway is a wrapper around two `b256` types, allowing for the representation of 64-byte values. It is defined as a struct:
 
-<<< @/../../docs-snippets/src/guide/types/bits512.test.ts#bits512-1{rust:line-numbers}
+```rs
```

Comment:
```suggestion
```sway
```

Could also put within a library in `docs-snippets2/sway/...`.

Reply:
@petertonysmith94 The Vitepress snippets plugin does not support the Sway language:
<img width="718" alt="Screenshot 2024-10-21 at 09 50 21" src="https://github.com/user-attachments/assets/bc301910-1e8c-4708-b886-39b13df0f32e">


using `rs`:
<img width="704" alt="Screenshot 2024-10-21 at 09 35 19" src="https://github.com/user-attachments/assets/2ed93beb-cccb-4f3c-b973-fe616ccb1194">

using `sw`: 
<img width="723" alt="Screenshot 2024-10-21 at 09 35 01" src="https://github.com/user-attachments/assets/1bd1f1d7-b63a-4a9f-8ca1-280b4302eff3">

This snippet was being hardcoded because it is a copy of the actual Sway `b512` implementation: https://github.com/FuelLabs/sway/blob/029c593b5f33a5c18ed2ddf91b1db12869139c91/sway-lib-std/src/b512.sw#L8-L11

I've added to a Sway project but I am unsure if that can mislead users to think that they need to declare this string to use it: https://github.com/FuelLabs/fuels-ts/pull/3340/commits/392eb2a18d20d0f93a9bd9d3a213b487dcfb00a9